docs/getting-started-guides/docker-multinode/common.sh

```diff
@@ -0,0 +1,437 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Utility functions for Kubernetes in docker setup
+# Authors @luxas @wizard_cxy @resouer @loopingz 
+
+# Variables
```

Comment:
I'm open to all kinds of solutions. Right now, there is no given place in the repo for this, so let's discuss about what would be the best way of doing it.

For context, see #13901

It would be _so_  much easier if we didn't have to restart `docker`. I think @mikedanese mentioned it may be possible to not have to (something like [this](https://github.com/kubernetes/kubernetes/issues/20355#issuecomment-186675564)?)